pkg/controller/discovery/web/pv.go

```diff
@@ -0,0 +1,75 @@
+package web
+
+import (
+	"encoding/json"
+	"github.com/fusor/mig-controller/pkg/controller/discovery/model"
+	"github.com/gin-gonic/gin"
+	"k8s.io/api/core/v1"
+	"net/http"
+)
+
+const (
+	PvsRoot = NamespaceRoot + "/persistentvolumes"
```

Comment:
Looks like a typo here, it should be 
```golang
PvsRoot = ClusterRoot + "/persistentvolumes"
```

Reply:
Oh, right.  I was wondering why the path was not what I expected when typing it :)

Reply:
Thanks :)

Reply:
Fixed.